syntax = "proto3";

package ServiceSuiteEx;

service ServiceEx
{
  rpc unaryRPC (RequestMsgEx) returns (ResponseMsgEx) {}
}

message RequestMsgEx {
  string foo = 1;
}

message ResponseMsgEx {
  string bar = 1;
}
